From 47b4077203548888216a32e38814ee3183588981 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Fri, 16 Apr 2021 12:55:22 +0200
Subject: [PATCH] Set MODULE_VERSION to 5.0.0

Without this the generated pkg-config files are invalid.

As the libraries are also called libQt5* by the build system, let's set
MODULE_VERSION to 5.0.0.
---
 .qmake.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.qmake.conf b/.qmake.conf
index 8840bd9..3c4fa5e 100644
--- a/.qmake.conf
+++ b/.qmake.conf
@@ -1,3 +1,3 @@
 load(qt_build_config)
 
-MODULE_VERSION = 0.0.0
+MODULE_VERSION = 5.0.0
-- 
2.31.1

